src/index.ts

```diff
@@ -101,11 +103,11 @@ const appActions = {
   },
 }
 
-function view(state, actions) {
+function view(state: State, actions: Actions) {
```

Comment:
A comment should be added here to describe this function, similar to render() and _renderNodes().